care/emr/migrations/0032_observationdefinition_qualified_ranges.py

```
@@ -0,0 +1,18 @@
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('emr', '0032_tokenbooking_associated_encounter'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='observationdefinition',
```

Ruff (0.12.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)